### PR TITLE
Lg/smart contract blockchain

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
**Overview**
Why do we need this feature? What are we trying to accomplish?

**Requirements**
What requirements are necessary to consider this problem solved?

**Considered Solutions**
What solutions were considered?

**Recommended Solution**
What solution are you recommending? Why?

**Considerations**
What impact will this change have on security, performance, users (e.g. breaking changes) or the team (e.g. maintenance costs)?

### CODE REVIEW

File: modules/ROOT/pages/http-middleware.adoc [hunk 1]
```diff
@@ -20,7 +20,7 @@ Migration to the new architecture doesn’t have to be completed immediately, bu
 Before we describe the new architecture, it’s useful to have some context about the old approach that we are replacing and how the code has worked up to this point. 
 Previously, developing a front-end on the Internet Computer involved adding a front-end assets canister with a `retrieve()` function. 
 The `retrieve()` function would take a path and return a blob. The blob returned by the `retrieve()` function would then have an `index.js` file that contained some JavaScript and static HTML called the `bootstrap` code. 
-After you deployed a canister on the Internet Computer, accessing the canister using the `<CANISTER_ID>.ic0.app` URL executed the `bootstrap` code, which in turn, performed the following steps:
+After you deployed a smart contract canister on the Internet Computer, accessing the canister using the `<CANISTER_ID>.ic0.app` URL executed the `bootstrap` code, which in turn, performed the following steps:
```
Comment (Eve832) [inline modules/ROOT/pages/http-middleware.adoc:23]: At what point do we introduce what a smart contract canister is? I might suggest that wherever we first introduce the idea of a canister, we talk about it in terms of an "extension" of smart contracts or something.

File: modules/developers-guide/pages/concepts/nodes-subnets.adoc [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 :platform: Internet Computer platform
 :IC: Internet Computer
 :company-id: DFINITY
 :sdk-short-name: DFINITY Canister SDK
 
-{IC} *sub-networks* provide physical hardware and resources—like CPU and memory—for performing software operations. Each sub-network consists of some number of individual machines—connected peer computers called *nodes*—that run the software components of the {IC} protocol. 
+{IC} *sub-networks* provide physical hardware and resources—like CPU and memory—for performing software operations. Each sub-network blockchain consists of some number of individual machines—connected peer computers called *nodes*—that run the software components of the {IC} protocol. 
 
-The {IC} software components that run on each node are called a *replica* because they replicate state and computation across all of the nodes in a sub-network.
+The {IC} software components that run on each node are called a *replica* because they replicate state and computation across all of the nodes in a sub-network blockchain.
```
Comment (Eve832) [inline modules/developers-guide/pages/concepts/nodes-subnets.adoc:11]: This sentence tells me that there is a separate blockchain for each sub-network. Is this true?

File: modules/developers-guide/pages/concepts/nodes-subnets.adoc [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 
 The core components of a replica are organized into the following logical layers:
 
-* A *peer-to-peer* (P2P) networking layer that collects and advertises messages from users, from other nodes in its sub-network, and from other sub-networks. Messages received by the peer-to-peer layer are replicated to all of the nodes in the sub-network to ensure the security, reliability, and resiliency of the platform.
+* A *peer-to-peer* (P2P) networking layer that collects and advertises messages from users, from other nodes in its sub-network blockchain, and from other sub-networks. Messages received by the peer-to-peer layer are replicated to all of the nodes in the sub-network to ensure the security, reliability, and resiliency of the platform.
```
Comment (Eve832) [inline modules/developers-guide/pages/concepts/nodes-subnets.adoc:15]: same as above

File: modules/developers-guide/pages/glossary.adoc [hunk 4]
```diff
@@ -12,7 +12,7 @@ actor::
 
 [[g-canister]]
 canister::
-  A canister is a conceptual object with a universally-unique identifier and an owner that defines the boundaries of a specific application, service, or micro-site. 
+  A canister is a conceptual object similar to a smart contract with a universally-unique identifier and an owner that defines the boundaries of a specific application, service, or micro-site.
```
Comment (Eve832) [inline modules/developers-guide/pages/glossary.adoc:15]: suggest taking this out of the glossary and adding it to the introduction of the canister?

Comment (Eve832) [inline modules/developers-guide/pages/glossary.adoc:15]:
```suggestion
  A smart contract canister is a conceptual object similar to a smart contract with a universally-unique identifier and an owner that defines the boundaries of a specific application, service, or micro-site. 
```

**Review by Eve832 — APPROVED** [review]

**Review by lsgunnlsgunn — COMMENTED** [review]
Added smart contract to the description for a bunch of file too

**Pull request merged**